docs/tutorials/typescript/digital-ocean-deployment/index.mdx

```diff
@@ -0,0 +1,283 @@
+    connection,
+    namespace: namespace,
+  });
+}
+```
+
+<!--SNIPEND-->
+
+**Temporal Worker**
+
+When developing locally, your Temporal Worker may look like this:
+
+<!--SNIPSTART typescript-slack-app-temporal-application-dev-worker-->
+
+[docs/tutorials/typescript/work-queue-slack-app/code/temporal-application/src/dev-worker.ts](https://github.com/temporalio/temporal-learning/blob/edu-2636-workqueue-slackbot-tut/docs/tutorials/typescript/work-queue-slack-app/code/temporal-application/src/dev-worker.ts)
+
+```ts
+import "dotenv/config";
+import path from "path";
+import {Worker, NativeConnection} from "@temporalio/worker";
+import * as activities from "./activities/index";
+
+async function run() {
+  try {
+    const worker = await Worker.create({
+      namespace: process.env.TEMPORAL_DEV_NAMESPACE || "",
+      workflowsPath: path.resolve(__dirname, "./workflows"),
+      activities,
+      taskQueue: `${process.env.ENV}-temporal-iq-task-queue`,
+    });
+
+    await worker.run();
+  } catch (err) {
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+run();
+```
+
+<!--SNIPEND-->
+
+When using Temporal Cloud, your Temporal Client should look like this:
+
+<!--SNIPSTART typescript-slack-app-temporal-application-cloud-worker-->
+
+[docs/tutorials/typescript/work-queue-slack-app/code/temporal-application/src/cloud-worker.ts](https://github.com/temporalio/temporal-learning/blob/edu-2636-workqueue-slackbot-tut/docs/tutorials/typescript/work-queue-slack-app/code/temporal-application/src/cloud-worker.ts)
+
+```ts
+import "dotenv/config";
+import path from "path";
+import {Worker, NativeConnection} from "@temporalio/worker";
+import * as activities from "./activities/index";
+
+async function run() {
+  try {
+    const key = Buffer.from(
+      process.env.TEMPORAL_CLOUD_PRIVATE_KEY || "",
+      "utf-8"
+    );
+    const cert = Buffer.from(process.env.TEMPORAL_CLOUD_PEM || "", "utf-8");
+    const connection = await NativeConnection.connect({
+      address: process.env.TEMPORAL_CLOUD_ADDRESS || "",
+      tls: {
+        clientCertPair: {
+          crt: cert,
+          key: key,
+        },
+      },
+    });
+
+    const worker = await Worker.create({
+      connection,
+      namespace: process.env.TEMPORAL_CLOUD_NAMESPACE || "",
+      workflowsPath: path.resolve(__dirname, "./workflows"),
+      activities,
+      taskQueue: `${process.env.ENV}-temporal-iq-task-queue`,
+    });
+
+    await worker.run();
+  } catch (err) {
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+run();
+```
+
+<!--SNIPEND-->
+
+## Use `pm2` to run your Worker {#use-pm2}
+
+```shell
+sudo npm install -g pm2
+```
+
+Change directory into your project and run your Temporal Worker with `pm2`.
+
+```shell
+pm2 start <your-app-entry>.ts --interpreter ts-node
+```
+
+## Conclusion
+
+You should now have a TypeScript based Temporal Application running on a Digital Ocean Droplet using Temporal Cloud as your Temporal Application's supervisor.
```

Comment:
See earlier notes about whether we need this to be DO-specific at all -- but I think we have a solid 5 steps and then a conclusion, after we've moved some stuff into prereqs.